test/inductor/test_max_autotune.py

```diff
@@ -231,12 +231,54 @@ def mm(a, b):
             Y = mm(a, b)
             torch.testing.assert_close(Y_compiled, Y)
 
+    @unittest.skipIf(not SM90OrLater, "need sm_90")
+    def test_max_autotune_cutlass_backend_simple_fusion(
+        self, dynamic: bool = False, max_autotune_gemm_backends: str = "CUTLASS"
+    ):
+        """
+        Test simple fusion that's not covered by specific lowering
+        """
+
+        if max_autotune_gemm_backends == "CUTLASS" and torch.version.hip:
+            return
+
+        # TODO: If I set this to False, I get a compilation error due to
+        # mismatched types at some point. If I use fp32 for everything, the Cutlass op generator
+        # creates no viable candidate ops that are EVT-capable.
+        torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction = True
+
+        def mm(a, b):
+            return (a @ b) * 3.0
```

Comment:
It doesn't get much simpler than that: We just want to fuse a "multiply with constant 3.0" but we use just EVT-capable operators for that, and use their fusion mechanism for it.